src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.SemanticChangeProcessor.cs

```diff
@@ -339,6 +353,20 @@ private class ProjectProcessor : IdleProcessor
                         _pendingWork = new Dictionary<ProjectId, Data>();
 
                         Start();
+
+                        AsyncProcessorTask.ContinueWith(
+                            _ =>
+                            {
+                                foreach (var (projectId, data) in _pendingWork)
```

Comment:
>projectId [](start = 46, length = 9)

consider using `_` instead of `projecTId` to make it clear it's not used.

Reply:
:arrow_right: Fixed in 9c28f4a